Microsoft.Packaging.Tools.Trimming/tasks/FileNode.cs

```diff
@@ -132,9 +133,94 @@ private void PopulateDependenciesInternal(IDictionary<string, FileNode> allFiles
             stack.Push(this);
 
             dependencies = new HashSet<FileNode>();
+            
+            if (File.Exists(SourceFile))
+            {
+                PolulateDependenicesFromMetadata(allFiles, preferNativeImage, log, stack);
```

Comment:
`Populate`?

Reply:
🤦‍♂️ doh.  Will fix.

Reply:
Not to pile on, but there are two typos.

Reply:
Yeah call site and definition, is there something else I am missing?

Reply:
I see it: Dependenices.  Oh my.  I must have been having a bad day.